src/g_game.c

```diff
@@ -610,7 +594,7 @@ void G_BuildTiccmd(ticcmd_t* cmd)
       y = FixedMul(FixedMul(y, y), y);
 
       y = direction[invert_look] * axis_look_sens * y / 10;
-      pitch -= FixedMul(lookspeed[0], y);
+      pitch -= FixedMul(LOOKSPEED, y);
```

Comment:
Padlook is flipped, so reverse the sign here: `+=`

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1389/commits/3a0850a561001406e1880b3a27d6ff893c43bc4e